webapp/channels/src/components/user_settings/security/user_settings_security.test.tsx

```diff
@@ -186,7 +190,7 @@ describe('components/user_settings/display/UserSettingsDisplay', () => {
             preventDefault: jest.fn(),
         };
         wrapper.setState({authorizedApps: apps});
-        (wrapper.instance() as UserSettingsSecurity).deauthorizeApp(event);
+        (wrapper.instance()).deauthorizeApp(event);
```

Comment:
These parenthesis (and in some other place in the file) are now redundant.
```suggestion
        wrapper.instance().deauthorizeApp(event);
```

Reply:
I'm used to auto formatting with prettier which is difficult to enable on your project due to too much noisy changes, sorry to have missed that.

I'll check the pipeline error thanks